doc/man7/provider-mac.pod

```diff
@@ -178,21 +178,12 @@ Gets flags associated with the MAC.
 
 =for comment We need to investigate if this is the right approach
 
-=item B<OSSL_MAC_PARAM_ALGORITHM> (utf8 string)
-
-Sets the name of the underlying algorithm to be used.
-It must name a suitable algorithm for the MAC that's being used.
-
-=item B<OSSL_MAC_PARAM_MD> (utf8 string)
+=item B<OSSL_MAC_PARAM_CIPHER> (utf8 string)
 
 =item B<OSSL_MAC_PARAM_DIGEST> (utf8 string)
 
-=item B<OSSL_MAC_PARAM_CIPHER> (utf8 string)
-
-These have the same meaning as B<OSSL_MAC_PARAM_ALGORITHM>, but specify
-the expected operation for the underlying algorithm.
-These are regarded as antiquated, but are kept for easier transition from
-legacy MAC implementations.
+Sets the name of the underlying cipher or digest to be used.
```

Comment:
This probably needs to be seperated now so that param_cipher has its own description.

Reply:
Why?  isn't "underlying cipher or digest" clear enough?

Reply:
I guess its ok..